create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils/common.py

```diff
@@ -36,6 +36,12 @@
             'port': trafaret.Int(),
         }),
     {%- endif %}
+{%- if cookiecutter.use_sentry == 'y' %}
+    trafaret.Key('sentry'):
+        trafaret.Dict({
+            'dsn': trafaret.String(),
+        }),
+    {%- endif %}
```

Comment:
After generation a new project with `sentry`, I will get error that my config file doesn't have `sentry` key. Am i right? If it's `true` so add this keys to `config` files, please.